src/shared/util.js

```diff
@@ -1329,6 +1493,64 @@ MessageHandler.prototype = {
     return capability.promise;
   },
   /**
+   * Sends a message to the comObj to invoke the action with the supplied data.
+   * Expect that other side will callback to signal start_complete.
+   * @param {String} actionName Action to call.
+   * @param {JSON} data JSON data to send.
+   * @param {Object} queueingStrategy strategy to signal backpressure based on
+   *                 internal queue.
+   * @param {Array} [transfers] Optional list of transfers/ArrayBuffers.
+   * @return {ReadableStream} ReadableStream to read data in chunks.
```

Comment:
> note that the other JSDoc comments in MessageHandler aren't entirely correct

I referred those comments to write this :)
Okay, i will do as you suggested, and also change other comments in `MessageHandler` to follow this.